arches_references/urls.py

```diff
@@ -10,53 +10,65 @@
     ListItemImageView,
     ListItemImageMetadataView,
     ListItemValueView,
+    PluginWithFullRedirectView,
 )
 
 urlpatterns = [
-    path("api/controlled_lists/", ListsView.as_view(), name="controlled_lists"),
+    # Override plugins routes from core to preserve full path on auth redirect.
     path(
-        "api/controlled_list/<uuid:list_id>/",
+        "plugins/<uuid:pluginid>/<path:path>",
+        PluginWithFullRedirectView.as_view(),
+        name="plugins",
+    ),
+    path(
+        "plugins/<slug:slug>/<path:path>",
+        PluginWithFullRedirectView.as_view(),
+        name="plugins",
+    ),
```

Comment:
Will this override bleed through to applications that have this as a dependency? If so, should this change be implemented in core?

Reply:
Good q. Whether it bleeds through is up to the application technically in how they include their urls, but in practice yeah it would probably bleed through.

This was here before I knew we were supporting 8.x only. Now that I know that, we should contribute this in core. 👍

Reply:
archesproject/arches#11325